Integrations/integration-Cisco-umbrella.yml

```diff
@@ -0,0 +1,885 @@
+           };
+        });
+
+        // build context
+        var domainContext = [];
+        var domainMaliciousContext = [];
+
+        // map matches to domains & check if matches is malare
+        matches.forEach(function (match) {
+            var securityCategoriesStr = match.securityCategories.join(',');
+            var domain = {
+                SecurityCategory: securityCategoriesStr,
+                FirstSeen: match.firstSeen,
+                FirstSeenISO:match.firstSeenISO
+            };
+            domainContext.push(domain);
+            if(securityCategoriesStr.indexOf('Malware') > -1) {
+                domainMaliciousContext.push(domain);
+            }
+        });
+
+        var context = {
+            Domain: domainContext
+        };
+
+        if(domainMaliciousContext.length > 0) {
+            context['Domain/Malicious'] = domainMaliciousContext;
+        }
+        return createTableEntry("Search Results:", table, context);
+    }
+
+    var getDomainCooccurrences = function(domain) {
+        domain = domainWithoutProtocol(domain);
+        var url = baseURL + '/recommendations/name/' + encodeURIComponent(domain) + '.json';
+        var res = sendRequest(url);
+
+        var occurrences = res && res.pfs2;
+        if (!occurrences || !(occurrences instanceof Array )) {
+            throw 'Got an illegal response\n.Response: ' + JSON.stringigy(res);
+        }
+
+        // table
+        var table = occurrences.map(function(occurrence) {
+           return {
+               Name: occurrence[0],
+               Score: occurrence[1]
+           }
+        });
+
+        // context
+        var context = {};
+        if (occurrences.length > 0) {
+            context.Domain = {
+                Name: domain,
+                CoOccurrences: table
+            };
+        }
+        return createTableEntry("Co-occurrences:", table, context);
+    }
+
+    var getDomainRelated = function(domain) {
+        domain = domainWithoutProtocol(domain);
+        var url = baseURL + '/links/name/' + encodeURIComponent(domainArg) + '.json';
+        var res = sendRequest(url);
+
+        var related = res && res.tb1;
+        if (!related || !(related instanceof Array )) {
+            throw 'Got an illegal response\n.Response: ' + JSON.stringigy(res);
+        }
+
+        // table
+        var table = related.map(function(r) {
+           return {
+               Name: r[0],
+               Score: r[1]
+           }
+        });
+
+        // context
+        var context = {};
+        if (related.length > 0) {
+            context.Domain = {
+                Name: domain,
+                Related: table
+            };
+        }
+        return createTableEntry("Related Domains:", table, context);
+    }
+
+    // This object describe the result of the http request of getDomainSecurity function
+    // each field has Name,Iinfo & ContextKey  - thats just looks scarry
+    var SecurityResultInfo = {
```

Comment:
put this declaration in the top of the code section and not between functions, or internally inside the function as a declaration